v0.1.4

Let's push v0.1.14 to pip and conda after finishing #42 and #62 . 

Unfortunately I don't think anyone has written a conda recipe yet for anndata, so we'll have to create one and push it along with ours.

I'm a bit confused about how to `bumpversion` and `tag`. I tried to `bumpversion batch` with or without `git tag v0.1.4`. For both cases, I got `fatal: tag 'v0.1.3' already exists`. 

I also tried to change `__version__ = '0.1.3' ` to `__version__ = '0.1.4'` and I got `AssertionError: Did not find '0.1.3' or '__version__ = '0.1.3'' in file scvi/__init__.py`. 

Do you remember what were the procedures you used to `bumpversion`? Thanks!!

--- a/scvi/__init__.py
+++ b/scvi/__init__.py
@@ -4,4 +4,4 @@
 
 __author__ = "Romain Lopez"
 __email__ = "romain_lopez@berkeley.edu"
-__version__ = '0.1.3'
+__version__ = '0.1.4'

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.1.3
+current_version = 0.1.4
 commit = True
 tag = True
 

--- a/setup.py
+++ b/setup.py
@@ -54,6 +54,6 @@ setup(
     test_suite='tests',
     tests_require=test_requirements,
     url='https://github.com/YosefLab/scVI',
-    version='0.1.3',
+    version='0.1.4',
     zip_safe=False,
 )

